kthfsdv_exc2.py

```diff
@@ -0,0 +1,72 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import sys
+import os
+
+
+class Function:
+    def __init__(self, plotting_mode='-d'):
+        self.plotting_mode = plotting_mode
+        os.system("clear")
+        if self.plotting_mode == '-d':
+            self.default_plot()
+        elif self.plotting_mode == '-s':
+            try:
+                self.static_plot()
+            except ValueError:
+                os.system("clear")
+                print('Wrong arguments passed')
+                print('Specify: first_number last_number number_of_samples')
+        elif self.plotting_mode == '-t':
+            self.time_plot()
+        else:
+            print('No such arguments')
+
+    def update(self, t):
+            h = lambda a: 3 * np.pi *\
+                np.exp(-1 * 5 * np.sin(2 * np.pi * 1 * a))
+            return h(self.t)
```

Comment:
This is correct but looks a bit redundant to define a lambda function if you are only returning the resulting value. it could be just:
```suggestion
    def update(self, t):
            return 3 * np.pi * np.exp(-1 * 5 * np.sin(2 * np.pi * 1 * t))
```

Reply:
It is a very minor thing anyway